sympy/stats/tests/test_joint_rv.py

```diff
@@ -54,6 +54,55 @@ def test_NormalGamma():
         3*sqrt(10)*gamma(S(7)/4)/(10*sqrt(pi)*gamma(S(5)/4))
     assert marginal_distribution(ng, y)(1) == exp(-S(1)/4)/128
 
+def test_GeneralizedMultivariateLogGammaDistribution():
+    from sympy.stats.joint_rv_types import GMVLG
+    from sympy import gamma
+    omega = Matrix([[1, 0.5, 0.5, 0.5],
+                     [0.5, 1, 0.5, 0.5],
+                     [0.5, 0.5, 1, 0.5],
+                     [0.5, 0.5, 0.5, 1]])
+    v, l, mu = (4, [1, 2, 3, 4], [1, 2, 3, 4])
+    y_1, y_2, y_3, y_4 = symbols('y_1:5', real=True)
+    n = symbols('n', negative=False, integer=True)
+    G = GMVLG('G', omega, v, l, mu)
+    den = "5*2**(2/3)*5**(1/3)*Sum(4*24**(-n - 4)*(-2**(2/3)*5**(1/3)/4 + 1)**n*"+\
```

Comment:
Better to use `("hello" "world")` instead of `+` for (constant) string concatenation.